docs/orchestrating-elastic-stack-applications/elasticsearch/users-and-roles.asciidoc

```diff
@@ -0,0 +1,154 @@
+:parent_page_id: elasticsearch-specification
+:page_id: users-and-roles
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{parent_page_id}.html#k8s-{page_id}[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Users and roles
+
+== Default elastic user
+
+A default user named `elastic` is created automatically, and is assigned the `superuser` role.
```

Comment:
Maybe it's obvious, but would it be worth to remind here *when* the default user is created?